### ISSUE TITLE
VirtualTable should update when height changes

### ISSUE THREAD

**Issue by freund17** [DevExpress/devextreme-reactive#748]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### Current Behaviour

Changing the height of VirtualTable doesn't update its contents.
Rows hidden because of a small height stay hidden when the height is increased.
After increasing the height scrolling is necessary to update the view.

### Expected Behaviour

Scrolling should not be necessary. Changing the height should suffice.

### Steps to Reproduce (for Bugs)

https://codesandbox.io/s/1yor2kwvkq

### Your Environment

devextreme-reactive: 1.0.0
react: 16.2.0
browser: Chrome 64.0.3282.167
bootstrap: none
react-bootstrap: none
material-ui: 1.0.0-beta.31

**Comment by freund17** [comment]
Something along the line of adding

```js
componentWillUpdate(nextProps) {
  const { height } = this.props

  if (height !== nextProps.height) {
    this.setState({
      visibleBodyRows: this.getVisibleRows()
    })
  }
}
```

to https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid/src/components/table-layout/virtual-table-layout.jsx
should do the trick. (untested)

**Comment by kvet** [comment]
Hi,
 
The fix is available in the 1.0.1 release.
